### PR TITLE
fixed mispelled hamburger-menu css style issu

### PR DESCRIPTION
**Pull request by serhiicss**
PR closes issue #42. Thank you.

### CODE REVIEW

File: public/assets/scss/header/_header.scss [hunk 1]
```diff
@@ -269,18 +269,19 @@
                     }
                 }
             }
-            .humberger-menu {
+            .hamburger-menu {
                 span {
                     &.text-white {
                         color: $heading-color !important;
+                        color: red !important;
```
Comment (QjStout) [inline public/assets/scss/header/_header.scss]: What is this line for? (ln:276)

**Review by QjStout — CHANGES_REQUESTED** [review]
Can you please fill out the submission template provided.
It's especially important to do the issue linking so that the issue you are resolving closes.

File: public/assets/scss/header/_header.scss (hunk 1 above)
Comment (QjStout) [reply public/assets/scss/header/_header.scss]: BTW, it is fine to leave off the `Advice for Reviewers & Testing Notes` and `Screenshots` sections for simple stories, but if doing so please delete those sections before submitting

**Comment by serhiicss** [comment]
I removed the extra line (I used it for testing and forgot to remove it initially). I mistakenly deleted the submission template and can't find it now :( sorry.

**Comment by QjStout** [comment]
That's ok @serhiicss
Just write `closes #42` so that the issue is linked to your PR

**Review by QjStout — APPROVED** [review]
lgtm 👍

**Pull request merged**